src/DssSpell.t.sol

```diff
@@ -491,24 +444,6 @@ contract DssSpellTest is DssSpellTestBase {
         hevm.warp(OCT_01_2022 + 31 days);
         assertTrue(tryVest(address(vest), 10));
         assertEq(dai.balanceOf(wallets.addr("DAIF_WALLET")), prevBalance + 67_863 * WAD);
-
-        vest.unrestrict(11);
-        prevBalance = dai.balanceOf(wallets.addr("DAIF_WALLET"));
-        hevm.warp(NOV_01_2022 + 304 days);
-        assertTrue(tryVest(address(vest), 11));
-        assertEq(dai.balanceOf(wallets.addr("DAIF_WALLET")), prevBalance + 329_192 * WAD);
-
-        vest.unrestrict(12);
-        prevBalance = dai.balanceOf(wallets.addr("DAIF_RESERVE_WALLET"));
-        hevm.warp(OCT_01_2022 + 92 days);
-        assertTrue(tryVest(address(vest), 12));
-        assertEq(dai.balanceOf(wallets.addr("DAIF_RESERVE_WALLET")), prevBalance + 270_000 * WAD);
-
-        vest.unrestrict(13);
-        prevBalance = dai.balanceOf(wallets.addr("ORA_WALLET"));
-        hevm.warp(JUL_01_2022 + 365 days);
-        assertTrue(tryVest(address(vest), 13));
-        assertEq(dai.balanceOf(wallets.addr("ORA_WALLET")), prevBalance + 2_337_804 * WAD);
     }
```

Comment:
I think you went a little hard on the delete here.  We should leave behind a complete example for the next person that needs this function.  This burned me on my duty, and I realized we didn't have a complete test for this anywhere.  This is that complete test, so either we should leave it around for the next person to modify, or we can slim it down to a minimal test.  Honestly, the later is probably too much work.  I would just make this test private and move on rather than deleting it.

Reply:
Done in 7a30e69